permissions/plugin-artifact-manager-s3.yml

```diff
@@ -9,6 +9,7 @@ developers:
 - "jglick"
 - "csanchez"
 - "oleg_nenashev"
+- "@company-cloudbees-developers"
```

Comment:
> Team company-cloudbees-developers not found

the team is called `cloudbees-developers`
https://github.com/jenkins-infra/repository-permissions-updater/blob/master/teams/cloudbees-developers.yml

Reply:
Yes my bad. I saw it the other day, but I left this fixing on the side as I wanted to get Oleg's clarification most importantly.
I'll fix it.

Reply:
```suggestion
- "@cloudbees-developers"
```

Reply:
It’s in all the files…